infra/build/functions/fuzzbench.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python3
+#
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+"""Does fuzzbench runs on Google Cloud Build."""
+
+import logging
+import os
+import sys
+
+import build_lib
+import build_project
+
+FUZZBENCH_BUILD_TYPE = 'coverage'
+FUZZBENCH_PATH = '/fuzzbench'
+
+
+def get_build_steps(  # pylint: disable=too-many-locals, too-many-arguments
+    project_name, project_yaml, dockerfile_lines, image_project,
+    base_images_project, config):
+  """Returns build steps for project."""
+  project = build_project.Project(project_name, project_yaml, dockerfile_lines,
+                                  image_project)
+  if project.disabled:
+    logging.info('Project "%s" is disabled.', project.name)
+    return []
+
+  config = build_project.Config(config.testing, None, config.repo,
+                                config.branch, config.parallel, config.upload)
+
+  # TODO(metzman): Make this a command line argument
+  fuzzing_engine = 'libafl'
```

Comment:
This seems pretty easy to pipe through. Should we just do this in the same PR?

Reply:
It's not trivial because we're basically outsourcing commandline arg parsing (and more importantly main) to another file